README.en.md

```diff
@@ -1,23 +1,23 @@
-# Spielsklave - you decide what I will develope next!
+# Spielsklave - You decide what I will develop next!
 
 ![Logo](img/md_logo.png)
 
 **Klicke [HIER](README.md) für die Deutsche Version.**
 
-Spielsklave (game slave) is a german [video series on YouTube](https://www.youtube.com/playlist?list=PL1td_Fr5vMGNqmdJOfnxDPKo_nO87Rs47).
-The audiences participate at surveys and decide what game I will develope.
+Spielsklave (lit. "game slave") is a German [video series on YouTube](https://www.youtube.com/playlist?list=PL1td_Fr5vMGNqmdJOfnxDPKo_nO87Rs47).
+The audience can decide how I will further develop this game by participating in surveys.
 
 ## The Game
 
-You can play the current game [here](https://letsgamedev.github.io/spielsklave/)!
+You can play the game in it's current state [here](https://letsgamedev.github.io/spielsklave/)!
```

Comment:
it's -> its
"You can play the game in it is current state" does not work here, hence the apostrophe needs to be removed.